environment.properties

```diff
@@ -1,7 +1,7 @@
 basepath = https://na132.lightning.force.com
 classicpath = https://na132.salesforce.com/home/home.jsp?source=lex
 loginpath = https://login.salesforce.com
-username = devsalesat09@example.com
-password = devsalesforce123
+username = mel@fundacion-jala.com
```

Comment:
Please put the correct email and password

Reply:
Solved.

Reply:
you have to put user devsalesat09@example.com and
pass devsalesforce123